pom.xml

```diff
@@ -26,30 +26,30 @@
 		<version>9</version>
 	</parent>
 	<properties>
-		<elasticsearch.version>7.2.1</elasticsearch.version>
+		<elasticsearch.version>7.17.1</elasticsearch.version>
 		<elasticsearch.plugin.classname>org.codelibs.elasticsearch.kuromoji.ipadic.neologd.KuromojiNeologdPlugin</elasticsearch.plugin.classname>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
-		<lucene.version>8.0.0</lucene.version>
-		<kuromoji-ipadic-neologd.version>20190808</kuromoji-ipadic-neologd.version>
+		<lucene.version>8.11.1</lucene.version>
+		<kuromoji-ipadic-neologd.version>20200910</kuromoji-ipadic-neologd.version>
```

Comment:
Hi! Thank you for your pr!
I also want to use this plugin in elasticsearch. I clone your branch but I found that I cannot get this part from maven.
maven reports that:
```
org.codelibs:lucene-analyzers-kuromoji-ipadic-neologd:jar:8.11.1-20200910 was not found in https://maven.codelibs.org/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of codelibs.org has elapsed or updates are forced
```

I also checked the maven repo and I found that there is no version for it..
https://maven.codelibs.org/org/codelibs/lucene-analyzers-kuromoji-ipadic-neologd/
 
Could you help to take a look?

Reply:
@boqigao 
Sorry for the late reply.
This kuromoji-ipadic-neologd is indeed not in maven repo, so I am using a locally generated version use [this](https://github.com/kazuhira-r/kuromoji-with-mecab-neologd-buildscript).

If you build it here with lucene version 8.11.1, it should generate a usable jar.

Reply:
Hi, @r4-keisuke  I build own es 7.17.1 version with locally `kuromoji-ipadic-neologd`
```
                <dependency>
                        <groupId>org.codelibs</groupId>
                        <artifactId>lucene-analyzers-kuromoji-ipadic-neologd</artifactId>
                        <version>8.11.1-20200910</version>
                        <scope>system</scope>
                        <systemPath>/path/to/jar_file/lucene-analyzers-kuromoji-ipadic-neologd-8.11.1-20200910.jar</systemPath>
                </dependency>
```
but when i install plugin in es show this error:

```
"stacktrace": ["java.lang.NoClassDefFoundError: org/apache/lucene/analysis/ja/JapaneseIterationMarkCharFilter",
```
 
how can i fix?